### ISSUE TITLE
What's the preferred way to add a DOM ready handler?

### ISSUE THREAD

**Issue by thanatos** [jquery/learn.jquery.com#747]
[learn.jquery.com's document ready page](https://learn.jquery.com/using-jquery-core/document-ready/) suggests,

> Experienced developers sometimes use the shorthand `$()` for `$( document ).ready()`. If you are writing code that people who aren't experienced with jQuery may see, it's best to use the long form.

However, [api.jquery.com's reference](http://api.jquery.com/ready/) suggests,

> jQuery offers several ways to attach a function that will run when the DOM is ready. All of the following syntaxes are equivalent:
>
> * `$( handler )`
> * `$( document ).ready( handler )`
> * `$( "document" ).ready( handler )`
> * `$( "img" ).ready( handler )`
> * `$().ready( handler )`
>
> As of jQuery 3.0, only the first syntax is recommended; the other syntaxes still work but are deprecated.

These suggestions feel at odds with each other. Given the deprecation notice, it seems like learn.jquery.com's suggestion is outdated.

**Comment by AurelioDeRosa** [comment]
Good catch, we didn't update the relevant text in the learn website. Do you want to submit a PR for this?
